### PR TITLE
add region override so metadata is not required.

### PR DESCRIPTION
**Pull request by DanMolz**
I have added an option to set the aws_region in configmap for use cases with metadata is not available.

### CODE REVIEW

File: ekscloudwatch.go [hunk 1]
```diff
@@ -24,23 +24,24 @@ type EKSAuditLogs struct {
...
-		log.Printf("Could not get EC2 instance information.")
-		return nil, err
-	}
+	var err error
 
-	log.Printf("AWS Instance ID: %s", instanceIdentity.InstanceID)
+	var region string
+	// if awsRegionOverride isn't set use it, otherwise try to autodetect it from the instance
+	if awsRegionOverride != "" {
+    region = awsRegionOverride
```
Comment (luca-sd) [inline ekscloudwatch.go]: please check indent of this block

File: deployment.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec:
         app: eks-cloudwatch
     spec:
       containers:
-        - image: sysdiglabs/k8sauditlogforwarder:ekscloudwatch-0.2
+        - image: sysdiglabs/k8sauditlogforwarder:ekscloudwatch-dev
```
Comment (luca-sd) [inline deployment.yaml]: Please leave it 0.2, when this is merged we'll do the release, thanks!

File: ekscloudwatch-config.yaml [hunk 3]
```diff
@@ -14,6 +14,7 @@ data:
   # Required: CloudWatch query filter
   cw_filter: '{ $.sourceIPs[0] != "::1" && $.sourceIPs[0] != "127.0.0.1" }'
 
-  # Optional: the EKS cluster name
-  # This can be omitted if the EC2 instance can perform the ec2:DescribeInstances action
-  cluster_name: ""
+  # Optional: both the EKS cluster name and region must be set
+  # This can be omitted if the EC2 instance can perform the ec2metadata and ec2:DescribeInstances action
+  cluster_name: "lab"
+  aws_region: "us-west"
```
Comment (luca-sd) [inline ekscloudwatch-config.yaml]: given the code below it looks like by default those should be kept empty

**Review by luca-sd — COMMENTED** [review]
minor comments

**Review by luca-sd — APPROVED** [review]
lgtm

**Comment by DanMolz** [comment]
@luca-sd - all fixed.

**Pull request merged**